doc-content/enterprise-only/openshift/secrets-employeerostering-create-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='secrets-employeerostering-create-proc']
+= Creating the secrets for Employee Rostering
+
+You must create an SSL certificate for HTTPS access to the Employee Rostering application and provide it to your OpenShift environment as a secret.
+
+.Procedure
+. Generate an SSL keystore with a private and public key for SSL encryption for the Employee Rostering application. In a production environment, generate a valid signed certificate that matches the expected URL of the application. Save the keystore in a file named `keystore.jks`. Record the name of the certificate and the password of the keystore file.
```

Comment:
Should this be broken down into a few steps? If it's just a side note for production environment only, then maybe line break after first sentence (starting "In a production environment" on a new line to break it up). Just a thought.

Reply:
Added a note to BXMSDOC-3905. Any changes to this procedure would need SME review which we can't get for this cycle, and would go into all OpenShift docs too, so it has to be handled as part of the upcoming OpenShift restructure.